std/heap.zig

```diff
@@ -565,6 +565,53 @@ pub fn StackFallbackAllocator(comptime size: usize) type {
     };
 }
 
+pub const NoErrorOutStream = std.io.OutStream(error{});
```

Comment:
How does one acquire a NoErrorOutStream? If I were to for example use `(try std.io.getStdErr()).outStream()` this outstream has errors.

Reply:
I've used this before....:
```diff
diff --git a/std/debug.zig b/std/debug.zig
index c5741f390..6425a7526 100644
--- a/std/debug.zig
+++ b/std/debug.zig
@@ -2339,7 +2339,15 @@ fn readILeb128(in_stream: var) !i64 {
 }
 
 /// This should only be used in temporary test programs.
-pub const global_allocator = &global_fixed_allocator.allocator;
+var NoErrorStdErrStream = std.heap.NoErrorOutStream {
+    .writeFn = struct {
+        pub fn writeFn(out_stream: *std.heap.NoErrorOutStream, bytes: []const u8) error{}!void {
+            const stderr = io.getStdErr() catch return;
+            stderr.write(bytes) catch return;
+        }
+    }.writeFn,
+};
+pub const global_allocator = &std.heap.LoggingAllocator.init(&global_fixed_allocator.allocator, &NoErrorStdErrStream).allocator;
```